modules/imaging_browser/test/TestPlan.md

```diff
@@ -1,44 +1,43 @@
+ [Partial Automation Testing]
+3. Test that all filters work. When the Site filter is empty, all sites with which the user is associated should be displayed. Every site should be displayed if the user has the `imaging_browser_view_allsites` permission.
+4. Test Clear Filters button.
+5. Test column table is sortable by headers.
+6. Ensure that the hyperlinks in the Links column are active and load the correct dataset.
+7. Add more modalities (from the Scan_type column of the `mri_scan_type` table) to the Configuration -> Imaging Modules -> Tabulated Scan Types field, and ensure that for each added modality, a new corresponding column shows up in the Imaging Browser table. (This requires back-end access)
 
-### B. ViewSession / Volume List
+### View Session / Volume List
+8. Sidebar:  
+   - Ensure that all links work, including the Download DICOM option. 
+   - Ensure that projects can customize (add/remove) their list of instruments that can be linked to from the Configuration -> Imaging Modules -> Imaging Browser Links to Insruments.
+9. Select an image using the checkbox above the volume previews. Click the "3D Only" and "3D + Overlay" buttons. These should load the Brain Browser module.
+10. Visit level QC controls (Pass/Fail, Pending, Visit Level Caveat) viewable to all, editable if and only if user has permission `imaging_browser_qc`.
+11. Save button appears if and only if user has permission `imaging_browser_qc`.
+12. Test Save button works.
+13. Verify Visit-level QC controls and comments can be deleted/unchecked/emptied and saved.
+14. Test Breadcrumb link back to Imaging Browser.
 
-8. Sidebar:  all links work, including the Download DICOM option. Ensure that projects can customize (add/remove) their list of instruments that can be linked to from the Configuration/Imaging Modules/Imaging Browser Links to Insruments
-9. 3d panel overlay etc - they work.  Add panel checkbox works. 3D only or 3D+Overlay loads files if at least one image exists and is selected
-10. "Visit Level Feedback" - pops up QC window (see section E below)
-11. Visit level QC controls (Pass/Fail, Pending, Visit Level Caveat) viewable to all, editable IFF permission imaging_browser_qc
-12. Save button appears IFF permission imaging_browser_qc
-13. Test Save button works 
-14. Verify Visit-level QC controls and comments can be deleted/unchecked/emptied and saved
-15. Test Breadcrumb link back to Imaging Browser
-
-### C. Main panel:  per acquisition:
-
-16. Files can be downloaded (links clickable) only IFF has permission. Ensure that DICOM downloads are
+### Main panel:  Per Acquisition
+15. Files can be downloaded (links clickable) if and only if user has permission ????. Ensure that DICOM downloads are
```

Comment:
@cmadjar could you tell me which permission is necessary for download ?

Reply:
I don't think there are any permission required to download the file. As long as you have permission to view the page, you can download the file. @nicolasbrossard Am I forgetting something here?

Reply:
@cmadjar @ ridz1208 You are correct Cecile: as long as you have permission to view the page, you can download the file.

Reply:
@nicolasbrossard whats the permission to view the page ?
imaging_browser_qc ?

Reply:
imaging_browser_view_site, imaging_browser_view_allsites, imaging_browser_phantom_allsites or imaging_browser_phantom_ownsite
 :)

Reply:
Persmission `imaging_browser_qc` is still used in the code but is not defined in `0000-00-01-Permission.sql`. Something's fishy....

Reply:
@nicolasbrossard https://github.com/aces/Loris/blob/a9e36f6785dd8cf8066e3b122515bbe3115acdad/SQL/0000-00-01-Permission.sql#L57

It seems to be on the 23.0-release branch

Reply:
@ridz1208 I think this module utilizes `htdocs/mri/jiv/get_file.php` which doesn't do any permission checks. it relies on the module to display links for users. So I think any of the four permissions cecile listed will allow a user to download files.